### PR TITLE
Add lifestyle 450 to palette

### PR DESCRIPTION
**Pull request by joecowton1**
## What is the purpose of this change?

From Trello:

Ben W spotted this colour is not in code.

He is using it for the journalism team project in the dark mode.

https://theguardian.design/2a1e5182b/p/75f7ca-dark-palette/t/655202

## What does this change?

-   Add colour to `pinks`
-   add pink to `lifestyle` palette

### CODE REVIEW

**Comment by sndrs** [comment]
is @akemitakagi down with this?

File: .changeset/tough-pots-sell.md
Comment (sndrs) [inline .changeset/tough-pots-sell.md:2]: given this is a new feature, i think it should be a minor?

```suggestion
'@guardian/source-foundations': minor
```

File: packages/@guardian/source-foundations/src/colour/palette.ts
Comment (sndrs) [inline packages/@guardian/source-foundations/src/colour/palette.ts:185]: can you add the to story? https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/colour/palette.stories.mdx

**Review by sndrs — CHANGES_REQUESTED** [review]

**Comment by joecowton1** [comment]
> is @akemitakagi down with this? all good if so

It was taken from a ticket in the engineering backlog - @akemitakagi is it ok to add?

https://trello.com/c/zRJr1RtX/865-add-lifestyle450-f37abc-to-the-dark-mode-colour-palette-in-code

File: packages/@guardian/source-foundations/src/colour/palette.ts
Comment (joecowton1) [reply packages/@guardian/source-foundations/src/colour/palette.ts:185]: From what I can tell it's been added automatically, unless I'm missing something.

This the `palette` story for this branch: https://5e933197d94d9c002275a15e-qhzeudcqit.chromatic.com/?path=/story/packages-source-foundations-palette--page

Screenshot 2022-07-14 at 10 57 49

Comment (sndrs) [reply packages/@guardian/source-foundations/src/colour/palette.ts:185]: oh sorry joe! it's more sophisticated than i thought :)

Comment (sndrs) [reply packages/@guardian/source-foundations/src/colour/palette.ts:185]: although if that's true, how come chromatic hasn't spotted it?

Comment (joecowton1) [reply packages/@guardian/source-foundations/src/colour/palette.ts:185]: Is it because when the story is snapshotted by Chromatic, the `lifestyle` palette is hidden?

Screenshot 2022-07-14 at 13 14 58

Comment (sndrs) [reply packages/@guardian/source-foundations/src/colour/palette.ts:185]: oh yeah of course! we should probably change that to show them all i guess?

Comment (joecowton1) [reply packages/@guardian/source-foundations/src/colour/palette.ts:185]: yeah i think so! I'll ticket that up and investigate - will be a longer page but defeats the object if we're hiding the palette

**Comment by joecowton1** [comment]
@sndrs are we ok to unblock / merge this one?

**Review by sndrs — APPROVED** [review]
looks great but let's merge #482 first? this will then also be a good test that the story has changed and needs confirming

**Comment by sndrs** [comment]
second thoughts, let's not block this on #482 – i think there's some more to do there but this is ready. it's great that it showed us that we needed to do #482 but this change is a good one to get out

**Pull request merged**
